assignments/arrays.js

```diff
@@ -62,33 +66,79 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 // }
 
 // ==== Challenge 1 ====
-// The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+// The dealer can't recall the information for a car with an id of 33 on his lot. 
+//Help the dealer find out which car has an id of 33 by logging 
+//the car's year, make, and model in the console log provided to you below:
+
+
+console.log("Car 33 is a " + inventory[32].car_model + " made by " + inventory[32].car_make + " and the year is " + inventory[32].car_year);
+
+
+//console.log("Car 33 is a" inventory[32] "*car year goes here* *car make goes here* *car model goes here*" );
 
 // ==== Challenge 2 ====
-// The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+// The dealer needs the information on the last car in their inventory.  
+//What is the make and model of the last car in the inventory?  Log the make and model into the console.
+
+let lastCar = inventory.length - 1;
+console.log(inventory[lastCar]);
```

Comment:
Good use of `.length`  reduced by one to target the last item, since arrays start at `0` index and not `1`